tasks/Windows/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+# tasks/Windows/main.yml: Deploy various client-side configurations for Sensu client on Windows
+
+    #- include_vars: "{{ ansible_distribution }}.yml"
+  - include_vars: "{{ ansible_os_family }}.yml"
+
+  - name: Ensure Temp Directory exists
+    win_file:
+      path: "{{ sensu_win_temp_path }}"
+      state: directory
+
+  - name: Download the Sensu installer msi into a local dir
+    win_get_url:
+      url: "{{ sensu_msi_download_path }}"
+      dest: "{{ sensu_win_temp_path }}\\sensu-installer.msi"
+      force: yes
```

Comment:
Do we need a force here? I believe this downloads the installer again every time we run the deployment?
I think we would only need to set this to true if the installer was changing.

Reply:
Maybe do a clever when here

Reply:
Is this something we should change within the scope of this current project?

Reply:
Can be fixed other time